src/go/configgenerator/route_generator_test.go

```diff
@@ -381,3 +415,13 @@ func TestMakeRouteConfigForCors(t *testing.T) {
 		}
 	}
 }
+
+// Used to generate a oversize cors origin regex or a oversize wildcard path regex.
+func getOversizeRegex() string {
```

Comment:
getOversizeUriTemplateForTest

Reply:
Rename it to be `getOverSizeRegexForTest` since it will also be used to generate `cors origin regex`.